internal/pkg/deploy/cloudformation/changeset.go

```diff
@@ -101,7 +102,7 @@ func (set *changeSet) execute() error {
 // createChangeSetOpt is a functional option to add additional settings to a CreateChangeSetInput.
 type createChangeSetOpt func(in *cloudformation.CreateChangeSetInput)
 
-func createChangeSetInput(stackName, templateBody string, options ...createChangeSetOpt) (*cloudformation.CreateChangeSetInput, error) {
+func createChangeSetInput(env *archer.Environment, templateBody string, options ...createChangeSetOpt) (*cloudformation.CreateChangeSetInput, error) {
```

Comment:
Can we keep this as stackName? The intent of `createChangeSetInput` is to be independent of an `environment` or `application` (I should have added comments :P)

Can we instead create a `createChangeSetOpt` that applies tags?
```go
// in changeset.go
func withEnvTags(env *archer.Environment) createChangeSetOpt {
	return func(in *cloudformation.CreateChangeSetInput) {
		in.tags = []*cloudformation.Tag{
                        {
			         Key:   aws.String("ecs-project"),
			         Value: aws.String(env.Project),
		        },
		       {
			         Key:   aws.String("ecs-env"),
			         Value: aws.String(env.Name),
		        },
	       }
	}
}
```
Then `DeployEnvironment` can use this function at l.54